alarm/alarm.py

```diff
@@ -1,58 +1,160 @@
+import argparse
+import os
 from pathlib import Path
+import multiprocessing
+import random
 import sys
 import time
 
-from decouple import config
-from playsound import playsound  # requires PyObjC (Mac)
+from dotenv import load_dotenv
+from playsound import playsound
 
-ALARM_MUSIC_FILE = config('ALARM_MUSIC_FILE')
-SECONDS_IN_MIN = 60
-POMODORO_TIMING = 25 * SECONDS_IN_MIN
+from . import __version__ as version
```

Comment:
__version__ is really common in Python community for representing the version of the module so I would not gave it another alias like `version` that looks like a normal variable and thus the reader expects it to hold something else than the module's version, right?

Reply:
ok we can use the dunder, happy with that, thanks